doc/source/workflow/overview.md

```diff
@@ -0,0 +1,208 @@
+# Overview of Seldon Core Components
+
+
+Seldon Core is build out of many components that deployed to Kubernetes works together as microservices.
+
+
+
+
+These are core Seldon's components:
+- Reusable and non-reusable [model servers](./overview.html#model-servers)
+- [Language Wrappers](./overview.html#language-wrappers) to containerise models
+- [SeldonDeployment](./overview.html#seldondeployment-crd) CRD and [Seldon Core Operator](./overview.html#seldon-core-operator)
+- [Service Orchestrator](./overview.html#service-orchestrator) for advanced inference graphs
+
+as well as integration with third-party systems:
+- Kubernetes Ingress integration with [Ambassador](https://www.getambassador.io/) and [Istio](https://istio.io/)
+- [Metrics](./overview.html#metrics-with-prometheus) with [Prometheus](https://prometheus.io/)
+- [Tracing](./overview.html#distributed-tracing-with-jaeger) with [Jaeger](https://www.jaegertracing.io/)
+- [Endpoint Documentation](./overview.html#endpoint-documentation) with [OpenApi](https://swagger.io/docs/specification/about/)
+
+Keep reading to learn more!
+
+
+![](../images/e2e-model-serving.svg)
+
+
+Usual e2e model serving with Seldon consists of few very important steps:
+1. *Data Scientist* prepare ML `model` using state of the art libraries (mlflow, dvc, xgboost, scikit-learn just to name a few).
+2. Trained model is uploaded to the central repository (e.g. S3 storage).
+3. *Software Engineer* prepare a `Model Server` which is uploaded as Docker image to the Image Registry.
+4. Deployment manifest (`SeldonDeployment` CRD) is created and applied to the Kubernetes cluster.
+5. Seldon Core `Operator` creates all required Kubernetes resources.
+6. Inference requests are served by Seldon Core `Orchestrator`.
+7. Metrics and tracing data is written to the third party systems.
```

Comment:
Maybe "Metrics and tracing data can be collected by leveraging our integration with third party frameworks"